apps/web-giddh/src/app/purchase/create-purchase-order/create-purchase-order.component.html

```diff
@@ -1660,6 +1661,10 @@
     [selectedAccountUniqueName]="selectedVendorForDetails" (onShortcutPress)="toggleAccountAsidePane()"
     (addEvent)="addNewSidebarAccount($event)" (updateEvent)="updateSidebarAccount($event)"></generic-aside-menu-account>
 
+<aside-menu-product-service *ngIf="asideMenuStateForProductService === 'in'" [class]="asideMenuStateForProductService"
```

Comment:
Is there any use of `[class]="asideMenuStateForProductService"` as no class is mentioned in class property that will get applied on truthy value of `asideMenuStateForProductService`

Reply:
in/out class gets applied on it.